packages/main/src/themes/Table.css

```diff
@@ -33,9 +34,9 @@
     box-sizing: border-box;
 }
 
-#busy-indicator {
+#loading {
     position: absolute;
-    z-index: 1;
     inset: 0;
     height: 100%;
+    z-index: 1;

```

Comment:
Maybe z-index: 2 is better here? When we have sticky headers there is already z-index: 1, so to avoid weird behavior 2 could avoid that